NoClassDefFoundError when releasing on Jira

```
test-db-builder-java % release-droid -lg java -n test-db-builder-java -g release     
2022-04-22 17:36:44.163 [INFO   ] Found an artifact on 'exasol/test-db-builder-java' repository. 
2022-04-22 17:36:45.606 [INFO   ] A GitHub workflow 'release_droid_print_quick_checksum.yml' has started. The Release Droid is monitoring its progress. This can take from a few minutes to a couple of hours depending on the build. 
2022-04-22 17:37:45.613 [INFO   ] Workflow is running for about 1 minutes. 
2022-04-22 17:38:26.980 [INFO   ] Checksum validation for 'exasol/test-db-builder-java' repository failed. 
2022-04-22 17:38:26.981 [INFO   ] Removing all artifacts from 'exasol/test-db-builder-java' repository. 
2022-04-22 17:38:28.148 [INFO   ] There are no artifacts on the 'exasol/test-db-builder-java' repository. 
2022-04-22 17:38:28.148 [INFO   ] Preparing a new artifact with a checksum. 
2022-04-22 17:38:28.531 [INFO   ] A GitHub workflow 'release_droid_prepare_original_checksum.yml' has started. The Release Droid is monitoring its progress. This can take from a few minutes to a couple of hours depending on the build. 
2022-04-22 17:39:28.537 [INFO   ] Workflow is running for about 1 minutes. 
2022-04-22 17:40:48.741 [INFO   ] Workflow is running for about 2 minutes. 
2022-04-22 17:41:49.555 [INFO   ] Workflow is running for about 3 minutes. 
2022-04-22 17:42:50.478 [INFO   ] Workflow is running for about 4 minutes. 
2022-04-22 17:43:51.181 [INFO   ] Workflow is running for about 5 minutes. 
2022-04-22 17:44:52.039 [INFO   ] Workflow is running for about 6 minutes. 
2022-04-22 17:45:52.874 [INFO   ] Workflow is running for about 7 minutes. 
2022-04-22 17:46:53.593 [INFO   ] Workflow is running for about 8 minutes. 
2022-04-22 17:47:54.339 [INFO   ] Workflow is running for about 9 minutes. 
2022-04-22 17:47:55.108 [INFO   ] Validation started. 
2022-04-22 17:47:56.637 [INFO   ] Releasing on MAVEN platform. 
2022-04-22 17:47:57.073 [INFO   ] A GitHub workflow 'release_droid_release_on_maven_central.yml' has started. The Release Droid is monitoring its progress. This can take from a few minutes to a couple of hours depending on the build. 
2022-04-22 17:48:57.079 [INFO   ] Workflow is running for about 1 minutes. 
2022-04-22 17:50:18.905 [INFO   ] Workflow is running for about 2 minutes. 
2022-04-22 17:50:19.642 [INFO   ] A MavenCentral release was published at: maven-central 
2022-04-22 17:50:19.841 [INFO   ] Release on platform MAVEN is finished! 
2022-04-22 17:50:19.841 [INFO   ] Validation started. 
2022-04-22 17:50:22.786 [INFO   ] Releasing on GITHUB platform. 
2022-04-22 17:50:23.925 [INFO   ] A GitHub workflow 'release_droid_upload_github_release_assets.yml' has started. The Release Droid is monitoring its progress. This can take from a few minutes to a couple of hours depending on the build. 
2022-04-22 17:51:23.928 [INFO   ] Workflow is running for about 1 minutes. 
2022-04-22 17:51:44.503 [INFO   ] A GitHub release was created at: https://github.com/exasol/test-db-builder-java/releases/tag/3.3.2 
2022-04-22 17:51:44.681 [INFO   ] Release on platform GITHUB is finished! 
2022-04-22 17:51:44.681 [INFO   ] Validation started. 
2022-04-22 17:51:44.868 [INFO   ] Releasing on JIRA platform. 
Exception in thread "main" java.lang.NoClassDefFoundError: org/springframework/beans/factory/DisposableBean
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:800)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:698)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at com.atlassian.jira.rest.client.internal.async.AsynchronousHttpClientFactory.createClient(AsynchronousHttpClientFactory.java:51)
        at com.atlassian.jira.rest.client.internal.async.AsynchronousJiraRestClientFactory.create(AsynchronousJiraRestClientFactory.java:36)
        at com.atlassian.jira.rest.client.internal.async.AsynchronousJiraRestClientFactory.createWithBasicHttpAuthentication(AsynchronousJiraRestClientFactory.java:42)
        at com.exasol.releasedroid.adapter.jira.JiraAPIAdapter.createJiraClient(JiraAPIAdapter.java:63)
        at com.exasol.releasedroid.adapter.jira.JiraAPIAdapter.createTicket(JiraAPIAdapter.java:39)
        at com.exasol.releasedroid.adapter.jira.JiraReleaseMaker.createTicketRequest(JiraReleaseMaker.java:52)
        at com.exasol.releasedroid.adapter.jira.JiraReleaseMaker.makeRelease(JiraReleaseMaker.java:37)
        at com.exasol.releasedroid.usecases.release.ReleaseInteractor.releaseOnPlatform(ReleaseInteractor.java:129)
        at com.exasol.releasedroid.usecases.release.ReleaseInteractor.releaseOnPlatforms(ReleaseInteractor.java:84)
        at com.exasol.releasedroid.usecases.release.ReleaseInteractor.makeRelease(ReleaseInteractor.java:66)
        at com.exasol.releasedroid.usecases.release.ReleaseInteractor.release(ReleaseInteractor.java:53)
        at com.exasol.releasedroid.main.ReleaseDroid.run(ReleaseDroid.java:63)
        at com.exasol.releasedroid.main.Runner.main(Runner.java:46)
Caused by: java.lang.ClassNotFoundException: org.springframework.beans.factory.DisposableBean
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        ... 22 more
```

--- a/.github/workflows/broken_links_checker.yml
+++ b/.github/workflows/broken_links_checker.yml
@@ -20,7 +20,7 @@ jobs:
         run: |
           mkdir -p ./target
           echo '{ "aliveStatusCodes": [429, 200] }' > ./target/broken_links_checker.json
-      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.13
         with:
           use-quiet-mode: 'yes'
           use-verbose-mode: 'yes'

--- a/.github/workflows/ci-build-next-java.yml
+++ b/.github/workflows/ci-build-next-java.yml
@@ -18,7 +18,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 17
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 17

--- a/.github/workflows/ci-build.yml
+++ b/.github/workflows/ci-build.yml
@@ -18,7 +18,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 11

--- a/.github/workflows/dependencies_check.yml
+++ b/.github/workflows/dependencies_check.yml
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 11

--- a/.github/workflows/release_droid_prepare_original_checksum.yml
+++ b/.github/workflows/release_droid_prepare_original_checksum.yml
@@ -12,7 +12,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 11
@@ -24,7 +24,7 @@ jobs:
       - name: Prepare checksum
         run: find target -maxdepth 1 -name *.jar -exec sha256sum "{}" + > original_checksum
       - name: Upload checksum to the artifactory
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
         with:
           name: original_checksum
           retention-days: 5

--- a/.github/workflows/release_droid_print_quick_checksum.yml
+++ b/.github/workflows/release_droid_print_quick_checksum.yml
@@ -12,7 +12,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 11

--- a/.github/workflows/release_droid_upload_github_release_assets.yml
+++ b/.github/workflows/release_droid_upload_github_release_assets.yml
@@ -16,7 +16,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 11

--- a/dependencies.md
+++ b/dependencies.md
@@ -27,81 +27,88 @@
 | [Hamcrest][27]                     | [BSD License 3][28]               |
 | [Maven Project Version Getter][29] | [MIT][16]                         |
 
+## Runtime Dependencies
+
+| Dependency         | License                          |
+| ------------------ | -------------------------------- |
+| [Spring Beans][31] | [Apache License, Version 2.0][1] |
+
 ## Plugin Dependencies
 
 | Dependency                                              | License                               |
 | ------------------------------------------------------- | ------------------------------------- |
-| [SonarQube Scanner for Maven][31]                       | [GNU LGPL 3][32]                      |
-| [Apache Maven Compiler Plugin][33]                      | [Apache License, Version 2.0][10]     |
-| [Apache Maven Enforcer Plugin][35]                      | [Apache License, Version 2.0][10]     |
-| [Maven Flatten Plugin][37]                              | [Apache Software Licenese][20]        |
-| [org.sonatype.ossindex.maven:ossindex-maven-plugin][39] | [ASL2][20]                            |
-| [Reproducible Build Maven Plugin][41]                   | [Apache 2.0][20]                      |
-| [Maven Surefire Plugin][43]                             | [Apache License, Version 2.0][10]     |
-| [Versions Maven Plugin][45]                             | [Apache License, Version 2.0][10]     |
-| [Apache Maven Assembly Plugin][47]                      | [Apache License, Version 2.0][10]     |
-| [Apache Maven JAR Plugin][49]                           | [Apache License, Version 2.0][10]     |
-| [OpenFastTrace Maven Plugin][51]                        | [GNU General Public License v3.0][52] |
-| [Project keeper maven plugin][53]                       | [The MIT License][54]                 |
-| [Artifact reference checker and unifier][55]            | [MIT][16]                             |
-| [Maven Failsafe Plugin][57]                             | [Apache License, Version 2.0][10]     |
-| [JaCoCo :: Maven Plugin][59]                            | [Eclipse Public License 2.0][60]      |
-| [error-code-crawler-maven-plugin][61]                   | [MIT][16]                             |
-| [Apache Maven Clean Plugin][63]                         | [Apache License, Version 2.0][10]     |
-| [Apache Maven Resources Plugin][65]                     | [Apache License, Version 2.0][10]     |
-| [Apache Maven Install Plugin][67]                       | [Apache License, Version 2.0][20]     |
-| [Apache Maven Deploy Plugin][69]                        | [Apache License, Version 2.0][20]     |
-| [Apache Maven Site Plugin][71]                          | [Apache License, Version 2.0][10]     |
+| [SonarQube Scanner for Maven][33]                       | [GNU LGPL 3][34]                      |
+| [Apache Maven Compiler Plugin][35]                      | [Apache License, Version 2.0][10]     |
+| [Apache Maven Enforcer Plugin][37]                      | [Apache License, Version 2.0][10]     |
+| [Maven Flatten Plugin][39]                              | [Apache Software Licenese][20]        |
+| [org.sonatype.ossindex.maven:ossindex-maven-plugin][41] | [ASL2][20]                            |
+| [Reproducible Build Maven Plugin][43]                   | [Apache 2.0][20]                      |
+| [Maven Surefire Plugin][45]                             | [Apache License, Version 2.0][10]     |
+| [Versions Maven Plugin][47]                             | [Apache License, Version 2.0][10]     |
+| [Apache Maven Assembly Plugin][49]                      | [Apache License, Version 2.0][10]     |
+| [Apache Maven JAR Plugin][51]                           | [Apache License, Version 2.0][10]     |
+| [OpenFastTrace Maven Plugin][53]                        | [GNU General Public License v3.0][54] |
+| [Project keeper maven plugin][55]                       | [The MIT License][56]                 |
+| [Artifact reference checker and unifier][57]            | [MIT][16]                             |
+| [Maven Failsafe Plugin][59]                             | [Apache License, Version 2.0][10]     |
+| [JaCoCo :: Maven Plugin][61]                            | [Eclipse Public License 2.0][62]      |
+| [error-code-crawler-maven-plugin][63]                   | [MIT][16]                             |
+| [Apache Maven Clean Plugin][65]                         | [Apache License, Version 2.0][10]     |
+| [Apache Maven Resources Plugin][67]                     | [Apache License, Version 2.0][10]     |
+| [Apache Maven Install Plugin][69]                       | [Apache License, Version 2.0][20]     |
+| [Apache Maven Deploy Plugin][71]                        | [Apache License, Version 2.0][20]     |
+| [Apache Maven Site Plugin][73]                          | [Apache License, Version 2.0][10]     |
 
 [15]: https://github.com/exasol/error-reporting-java
 [5]: https://www.opensource.org/licenses/mit-license.php
 [20]: http://www.apache.org/licenses/LICENSE-2.0.txt
-[43]: https://maven.apache.org/surefire/maven-surefire-plugin/
+[45]: https://maven.apache.org/surefire/maven-surefire-plugin/
 [16]: https://opensource.org/licenses/MIT
 [23]: https://github.com/mockito/mockito
 [29]: https://github.com/exasol/maven-project-version-getter
-[45]: http://www.mojohaus.org/versions-maven-plugin/
-[53]: https://github.com/exasol/project-keeper/
+[47]: http://www.mojohaus.org/versions-maven-plugin/
+[55]: https://github.com/exasol/project-keeper/
 [28]: http://opensource.org/licenses/BSD-3-Clause
-[33]: https://maven.apache.org/plugins/maven-compiler-plugin/
-[65]: https://maven.apache.org/plugins/maven-resources-plugin/
-[51]: https://github.com/itsallcode/openfasttrace-maven-plugin
-[63]: https://maven.apache.org/plugins/maven-clean-plugin/
-[60]: https://www.eclipse.org/legal/epl-2.0/
-[32]: http://www.gnu.org/licenses/lgpl.txt
+[35]: https://maven.apache.org/plugins/maven-compiler-plugin/
+[67]: https://maven.apache.org/plugins/maven-resources-plugin/
+[53]: https://github.com/itsallcode/openfasttrace-maven-plugin
+[65]: https://maven.apache.org/plugins/maven-clean-plugin/
+[62]: https://www.eclipse.org/legal/epl-2.0/
+[34]: http://www.gnu.org/licenses/lgpl.txt
 [1]: https://www.apache.org/licenses/LICENSE-2.0
-[59]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
+[61]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
 [24]: https://github.com/mockito/mockito/blob/main/LICENSE
-[41]: http://zlika.github.io/reproducible-build-maven-plugin
+[43]: http://zlika.github.io/reproducible-build-maven-plugin
 [8]: http://www.opensource.org/licenses/mit-license.php
-[31]: http://sonarsource.github.io/sonar-scanner-maven/
+[33]: http://sonarsource.github.io/sonar-scanner-maven/
 [19]: https://bitbucket.org/snakeyaml/snakeyaml
 [21]: https://junit.org/junit5/
-[37]: https://www.mojohaus.org/flatten-maven-plugin/flatten-maven-plugin
+[39]: https://www.mojohaus.org/flatten-maven-plugin/flatten-maven-plugin
 [2]: https://docs.atlassian.com/fugue-parent/4.1.0/apidocs/io/atlassian/fugue/package-summary.html
 [27]: http://hamcrest.org/JavaHamcrest/
 [7]: http://www.slf4j.org
 [4]: https://github-api.kohsuke.org/
-[55]: https://github.com/exasol/artifact-reference-checker-maven-plugin
-[49]: https://maven.apache.org/plugins/maven-jar-plugin/
+[57]: https://github.com/exasol/artifact-reference-checker-maven-plugin
+[51]: https://maven.apache.org/plugins/maven-jar-plugin/
 [13]: https://maven.apache.org/ref/3.8.5/maven-model/
 [3]: http://www.apache.org/licenses/LICENSE-2.0
 [17]: https://github.com/commonmark/commonmark-java/commonmark
 [9]: https://commons.apache.org/proper/commons-cli/
-[57]: https://maven.apache.org/surefire/maven-failsafe-plugin/
-[54]: https://github.com/exasol/project-keeper/blob/main/LICENSE
-[71]: https://maven.apache.org/plugins/maven-site-plugin/
-[52]: https://www.gnu.org/licenses/gpl-3.0.html
+[59]: https://maven.apache.org/surefire/maven-failsafe-plugin/
+[56]: https://github.com/exasol/project-keeper/blob/main/LICENSE
+[73]: https://maven.apache.org/plugins/maven-site-plugin/
+[54]: https://www.gnu.org/licenses/gpl-3.0.html
 [18]: https://opensource.org/licenses/BSD-2-Clause
 [10]: https://www.apache.org/licenses/LICENSE-2.0.txt
-[35]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
+[37]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
 [22]: https://www.eclipse.org/legal/epl-v20.html
-[67]: http://maven.apache.org/plugins/maven-install-plugin/
-[39]: https://sonatype.github.io/ossindex-maven/maven-plugin/
+[31]: https://github.com/spring-projects/spring-framework
+[69]: http://maven.apache.org/plugins/maven-install-plugin/
+[41]: https://sonatype.github.io/ossindex-maven/maven-plugin/
 [6]: https://www.eclipse.org/jgit/
-[69]: http://maven.apache.org/plugins/maven-deploy-plugin/
+[71]: http://maven.apache.org/plugins/maven-deploy-plugin/
 [11]: https://github.com/douglascrockford/JSON-java
 [0]: https://ecosystem.atlassian.net/wiki/spaces/JRJC/overview
-[61]: https://github.com/exasol/error-code-crawler-maven-plugin
+[63]: https://github.com/exasol/error-code-crawler-maven-plugin
 [12]: http://json.org/license.html
-[47]: https://maven.apache.org/plugins/maven-assembly-plugin/
+[49]: https://maven.apache.org/plugins/maven-assembly-plugin/

--- a/doc/changes/changes_1.0.0.md
+++ b/doc/changes/changes_1.0.0.md
@@ -10,6 +10,7 @@ Code name: Support for multi maven module projects
 
 * #223: Removed Spring Beans dependency with CVE-2022-22965
 * #222: Fixed executing the JAR by adding a manifest
+* #226: Fixed error when running Jira release
 
 ## Dependency Updates
 
@@ -21,27 +22,31 @@ Code name: Support for multi maven module projects
 * Updated `org.commonmark:commonmark:0.18.0` to `0.18.2`
 * Updated `org.eclipse.jgit:org.eclipse.jgit:5.13.0.202109080827-r` to `6.1.0.202203080745-r`
 * Updated `org.json:json:20210307` to `20220320`
-* Updated `org.kohsuke:github-api:1.301` to `1.305`
+* Updated `org.kohsuke:github-api:1.301` to `1.306`
 * Updated `org.slf4j:slf4j-simple:1.7.32` to `1.7.36`
 * Updated `org.yaml:snakeyaml:1.29` to `1.30`
+
+### Runtime Dependency Updates
+
+* Added `org.springframework:spring-beans:5.3.19`
 
 ### Test Dependency Updates
 
 * Added `com.exasol:maven-project-version-getter:1.1.0`
 * Updated `org.junit.jupiter:junit-jupiter:5.8.1` to `5.8.2`
-* Updated `org.mockito:mockito-core:4.1.0` to `4.5.0`
-* Updated `org.mockito:mockito-junit-jupiter:4.1.0` to `4.5.0`
+* Updated `org.mockito:mockito-core:4.1.0` to `4.5.1`
+* Updated `org.mockito:mockito-junit-jupiter:4.1.0` to `4.5.1`
 
 ### Plugin Dependency Updates
 
 * Added `com.exasol:artifact-reference-checker-maven-plugin:0.4.0`
 * Updated `com.exasol:error-code-crawler-maven-plugin:0.7.1` to `1.1.0`
-* Updated `com.exasol:project-keeper-maven-plugin:1.3.4` to `2.3.0`
+* Updated `com.exasol:project-keeper-maven-plugin:1.3.4` to `2.3.1`
 * Updated `io.github.zlika:reproducible-build-maven-plugin:0.14` to `0.15`
 * Updated `org.apache.maven.plugins:maven-clean-plugin:3.1.0` to `3.2.0`
 * Updated `org.apache.maven.plugins:maven-compiler-plugin:3.8.1` to `3.10.1`
 * Added `org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M6`
-* Updated `org.apache.maven.plugins:maven-site-plugin:3.9.1` to `3.11.0`
+* Updated `org.apache.maven.plugins:maven-site-plugin:3.9.1` to `3.12.0`
 * Updated `org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M4` to `3.0.0-M6`
 * Added `org.codehaus.mojo:flatten-maven-plugin:1.2.7`
 * Updated `org.itsallcode:openfasttrace-maven-plugin:1.2.0` to `1.5.0`

--- a/pk_generated_parent.pom
+++ b/pk_generated_parent.pom
@@ -9,6 +9,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
+        <test.excludeTags/>
     </properties>
     <licenses>
         <license>
@@ -128,6 +129,7 @@
                     <!-- Set the highest log level for coverage testing, so that we
           have a chance to reach branches in the logging lambdas too. -->
                     <argLine>-Djava.util.logging.config.file=src/test/resources/logging.properties ${argLine}</argLine>
+                    <excludedGroups>${test.excludeTags}</excludedGroups>
                 </configuration>
             </plugin>
             <plugin>
@@ -202,6 +204,7 @@
           <property>	                      in the logging lambdas too. -->
                     <argLine>-Djava.util.logging.config.file=src/test/resources/logging.properties ${argLine}</argLine>
                     <systemPropertyVariables/>
+                    <excludedGroups>${test.excludeTags}</excludedGroups>
                 </configuration>
                 <executions>
                     <execution>

--- a/pom.xml
+++ b/pom.xml
@@ -10,7 +10,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
-        <mockito.version>4.5.0</mockito.version>
+        <mockito.version>4.5.1</mockito.version>
     </properties>
     <repositories>
         <repository>
@@ -23,21 +23,15 @@
             <groupId>com.atlassian.jira</groupId>
             <artifactId>jira-rest-java-client-core</artifactId>
             <version>5.2.4</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.codehaus.jackson</groupId>
-                    <artifactId>jackson-mapper-asl</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.springframework</groupId>
-                    <artifactId>spring-core</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.springframework</groupId>
-                    <artifactId>spring-beans</artifactId>
-                </exclusion>
-            </exclusions>
         </dependency>
+        <!-- Update spring-beans version required by jira client to fix security issues
+             CVE-2022-22965, CVE-2016-1000027, CVE-2021-22118, CVE-2021-22096, CVE-2022-22950, CVE-2022-22965 -->
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-beans</artifactId>
+                <version>5.3.19</version>
+                <scope>runtime</scope>
+            </dependency>
         <dependency>
             <groupId>io.atlassian.fugue</groupId>
             <artifactId>fugue</artifactId>
@@ -46,7 +40,7 @@
         <dependency>
             <groupId>org.kohsuke</groupId>
             <artifactId>github-api</artifactId>
-            <version>1.305</version>
+            <version>1.306</version>
         </dependency>
         <dependency>
             <groupId>org.eclipse.jgit</groupId>
@@ -189,7 +183,7 @@
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>project-keeper-maven-plugin</artifactId>
-                <version>2.3.0</version>
+                <version>2.3.1</version>
                 <executions>
                     <execution>
                         <goals>
@@ -234,7 +228,7 @@
                 </plugin>
                 <plugin>
                     <artifactId>maven-site-plugin</artifactId>
-                    <version>3.11.0</version>
+                    <version>3.12.0</version>
                 </plugin>
                 <plugin>
                     <groupId>com.exasol</groupId>

--- a/src/main/java/com/exasol/releasedroid/adapter/jira/JiraAPIAdapter.java
+++ b/src/main/java/com/exasol/releasedroid/adapter/jira/JiraAPIAdapter.java
@@ -6,9 +6,7 @@ import java.net.URISyntaxException;
 
 import com.atlassian.jira.rest.client.api.JiraRestClient;
 import com.atlassian.jira.rest.client.api.JiraRestClientFactory;
-import com.atlassian.jira.rest.client.api.domain.BasicIssue;
-import com.atlassian.jira.rest.client.api.domain.IssueType;
-import com.atlassian.jira.rest.client.api.domain.Project;
+import com.atlassian.jira.rest.client.api.domain.*;
 import com.atlassian.jira.rest.client.api.domain.input.IssueInput;
 import com.atlassian.jira.rest.client.api.domain.input.IssueInputBuilder;
 import com.atlassian.jira.rest.client.internal.async.AsynchronousJiraRestClientFactory;
@@ -55,7 +53,7 @@ public class JiraAPIAdapter implements JiraGateway {
         }
     }
 
-    private JiraRestClient createJiraClient() throws JiraException {
+    JiraRestClient createJiraClient() throws JiraException {
         final String username = this.propertyReader.readProperty(JIRA_USERNAME_KEY, false);
         final String password = this.propertyReader.readProperty(JIRA_PASSWORD_KEY, true);
         final JiraRestClientFactory factory = new AsynchronousJiraRestClientFactory();

--- a/src/test/java/com/exasol/releasedroid/adapter/jira/JiraAPIAdapterTest.java
+++ b/src/test/java/com/exasol/releasedroid/adapter/jira/JiraAPIAdapterTest.java
@@ -1,0 +1,41 @@
+package com.exasol.releasedroid.adapter.jira;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.mockito.Mockito.lenient;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.atlassian.jira.rest.client.api.JiraRestClient;
+import com.exasol.releasedroid.usecases.PropertyReader;
+
+@ExtendWith(MockitoExtension.class)
+class JiraAPIAdapterTest {
+
+    @Mock
+    private PropertyReader propertyReaderMock;
+    private JiraAPIAdapter jiraAPIAdapter;
+
+    @BeforeEach
+    void setup() {
+        lenient().when(propertyReaderMock.readProperty("jira_username", false)).thenReturn("wrong-user");
+        lenient().when(propertyReaderMock.readProperty("jira_password", false)).thenReturn("wrong-password");
+        jiraAPIAdapter = new JiraAPIAdapter(propertyReaderMock);
+    }
+
+    @Test
+    void testCreatingJiraClientSucceeds() throws JiraException {
+        assertDoesNotThrow(() -> jiraAPIAdapter.createJiraClient());
+    }
+
+    @Test
+    void testCreatingJiraClientReturnsClient() throws JiraException {
+        final JiraRestClient client = jiraAPIAdapter.createJiraClient();
+        assertThat(client, notNullValue());
+    }
+}

